### PR TITLE
general admin fixes after checking

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixes some alignment, button height, input field widths after checking admin

### CODE REVIEW

File: djangocms_admin_style/sass/components/_content.scss [hunk 1]
```diff
@@ -235,11 +236,12 @@ ul.messagelist + #content {
     box-shadow: none !important;
     .deletelink-box {
         margin: 0;
-    }
-    .deletelink {
-        display: block;
-        @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
-        @include icon(trash);
+        .deletelink {
+           @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
```
Comment (ruslanmalogulko) [inline djangocms_admin_style/sass/components/_content.scss]: wrong indentation here

**Comment by Brankub** [comment]
LGTM

File: djangocms_admin_style/sass/components/_changelist.scss [hunk 2]
```diff
@@ -92,7 +92,7 @@
                 .noItemsRow {
                     th,
                     td {
-                       padding: 12px 0 12px 15px !important;
+                       padding: 10px 0 10px 15px !important;
```
Comment (ruslanmalogulko) [inline djangocms_admin_style/sass/components/_changelist.scss]: wrong indentation

Comment (Brankub) [inline djangocms_admin_style/sass/components/_changelist.scss]: catch! :)

**Comment by ruslanmalogulko** [comment]
:+1:

**Pull request merged**